cmd/controller-manager/main.go

```diff
@@ -101,11 +102,20 @@ func main() {
 	monitoringEnabled := os.Getenv("ENABLE_MONITORING")
 
 	if monitoringEnabled == "true" {
-		log.Info("Installing monitoring resources")
-
+		// Attempt to install monitoring resources on operator start
 		err = installMonitoring(ctx, serverClient)
+
+		ticker := time.NewTicker(5 * time.Second)
+
+		// Attempt to install monitoring resources every 5 minutes
```

Comment:
Just a minor improvement: how about only creating the ticker if installMonitoring fails and complete/cancel it once it succeeds (inside the ticker loop)?

Reply:
@lulf can do. I've just noticed that the ticker time also needs to updated to 5 minutes.